website/src/middlewares/locals.js

```diff
@@ -29,23 +53,18 @@ module.exports = function(req, res, next) {
       args.push(language.code);
       return shared.i18n.t.apply(null, args);
     },
-    siteVersion: forceRefresh.siteVersion,
-    Content: shared.content,
-    mods: require('../models/user').mods,
-    tavern: tavern, // for world boss
+    // Defined here and not outside of the middleware because tavern might be an
+    // empty object until the query to fetch it finishes
     worldDmg: (tavern && tavern.quest && tavern.quest.extra && tavern.quest.extra.worldDmg) || {},
-    _: _,
-    AMAZON_PAYMENTS: {
-      SELLER_ID: nconf.get('AMAZON_PAYMENTS:SELLER_ID'),
-      CLIENT_ID: nconf.get('AMAZON_PAYMENTS:CLIENT_ID')
-    }
   });
 
   // Put query-string party (& guild but use partyInvite for backward compatibility)
   // invitations into session to be handled later
-  try{
-    req.session.partyInvite = JSON.parse(utils.decrypt(req.query.partyInvite));
-  } catch(e){}
+  if(req.query.partyInvite){
+    try{
+      req.session.partyInvite = JSON.parse(utils.decrypt(req.query.partyInvite));
+    } catch(e){}
```

Comment:
I was unsure on wheter we could remove the try/catch at all since I added that `if`. I think it's there because `utils.decrypt` and `JSON.parse` throws an error and would break everything if for any reason a user passed an invalid query string